reliability/single-fit/logs/src/test/kotlin/com/datadog/android/logs/integration/LoggerTest.kt

```diff
@@ -703,6 +704,38 @@ class LoggerTest {
         assertThat(event0.getString("error.stack")).isEqualTo(fakeErrorStack)
     }
 
+    @RepeatedTest(16)
```

Comment:
do you need a repeated test here ?

Reply:
Following the pattern of the file. I think the idea is to do a lot of tests against random data in the integration phase.